### DISCUSSION TITLE
Analyzer suggestions are not reported at build-time

### DISCUSSION THREAD

**Discussion by Arthri** [dotnet/roslyn#72397] [Q&A]
Minimal reproduction: https://github.com/Arthri/dotnet-demo-roslyn-messages-severity

Analyzer errors/warnings are reported at build time, but analyzer suggestions/information aren't: where do they go?

**Comment by huoyaoyuan** [comment]
Does anything other than IDE shows these severities?

**Reply by Arthri** [reply]
If I recall correctly, `dotnet format` should show them when asked to, but I'm not aware of any other tools/systems which show them
